src/reducers/SelectedRevisions.jsx

```diff
@@ -0,0 +1,89 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = {
+  revisions: [
+    {
+      id: 1,
+      revision: 'coconut',
+      author: 'john@python.com',
+      push_timestamp: 42,
+      repository_id: 4,
+      revisions: [
+        {
+          revision: 'coconut',
+          author: 'john@python.com',
+          comments: 'This is my first message',
+        },
+      ],
+    },
+    {
+      id: 2,
+      revision: 'coconut 2',
+      author: 'johncleese@python.com',
+      push_timestamp: 43,
+      repository_id: 3,
+      revisions: [
+        {
+          revision: 'coconut 2',
+          author: 'johncleese@python.com',
+          comments: 'This is the second message',
+        },
+      ],
+    },
+    // {
+    //   id: 0,
+    //   projectName: 'moxilla central',
+    //   revision: '35g4bjh54jk345b',
+    //   author: 'author@mozila.com',
+    //   commitMessage: 'Added details for',
+    //   timeStamp: '00:00:00UTC',
+    // },
+    // {
+    //   id: 1,
+    //   projectName: 'raptor 1',
+    //   revision: '35g4bjh54jk345b',
+    //   author: 'author@mozila.com',
+    //   commitMessage: 'Added details for',
+    //   timeStamp: '00:01:00UTC',
+    // },
+    // {
+    //   id: 2,
+    //   projectName: 'auto',
+    //   revision: '35g4bjh54jk345b',
+    //   author: 'author@mozila.com',
+    //   commitMessage: 'Added details for',
+    //   timeStamp: '00:02:00UTC',
+    // },
+    // {
+    //   id: 3,
+    //   projectName: 'raptor 2',
+    //   revision: '35g4bjh54jk345b',
+    //   author: 'author@mozila.com',
+    //   commitMessage: 'Added details for',
+    //   timeStamp: '00:01:00UTC',
+    // },
+  ],
+};
+
+const compareRevisions = createSlice({
+  name: 'compareRevisions',
+  initialState,
+  reducers: {
+    resetState(state) {
```

Comment:
if you add an `afterEach` that calls `resetState` it should show as covered by tests